Array/C/princy9293_Find-largest-element.c

```diff
@@ -0,0 +1,24 @@
+//program to find largest element in array
+#include<stdio.h>
+#include<conio.h>
+int main()
+{
+	int a[100],i,n;
+	printf("enter the number of elements in array:\t");
+	scanf("%d",&n);
+	for(i=0;i<n;i++)
+	{
+	printf("enter element no. %d:\t",i+1);
+	scanf("%d",&a[i]);
+	}
+	for(i=1;i<=n;i++)      //to find largest number
+	{
+		if(a[0]<a[i])
+		{
```

Comment:
Since it is a single line code you can remove the brackets {} from line 17 and 19